Sources/CodegenCLI/Protocols/LogLevelSetter.swift

```diff
@@ -0,0 +1,18 @@
+import ApolloCodegenLib
+
+public protocol LogLevelSetter {
```

Comment:
This exists as a way to stub and mock the global state of `CodegenLogger` vs. implementing a protocol for the functionality of the logger itself.